.github/pull_request_template.md

```diff
@@ -1,31 +1,34 @@
 <!-- INSTRUCTIONS: 
-- PLEASE READ/FOLLOW THE DIRECTIONS IN EACH SECTION
+- READ/FOLLOW THE DIRECTIONS IN EACH SECTION
 - Complete the 'Commit Queue Requirements' below
-- Please use github markup as much as possible (https://docs.github.com/en/get-started/writing-on-github)
-- Please leave your PR in a draft state until all underlying work is completed.
+- Use GitHub markup as much as possible (https://docs.github.com/en/get-started/writing-on-github)
+- Leave your PR in a draft state until all underlying work is completed.
 -->
+
 ## Commit Queue Requirements:
 <!--
-- Please complete the items that follow this.
-- Please "check off" completed items. Use [X] for a filled in checkbox or leave it [ ] for an empty checkbox
-- Your PR will not be considered until all requirements are met.
+- Check off completed items. Use [X] for a filled in checkbox or leave it [ ] for an empty checkbox
+- Your pull request (PR) will not be considered until all requirements are met.
 - THIS IS YOUR RESPONSIBILITY
  -->
+- [ ] This PR addresses a relevant WM issue (if not, create an issue). 
+- [ ] All subcomponent pull requests (if any) have been reviewed by their code managers.
+- [ ] Run the full Intel+GNU RT suite (compared to current baselines), preferably on Ursa (Derecho or Hercules are acceptable alternatives). **Exceptions:** documentation-only PRs, CI-only PRs, etc.
+   - [ ] Commit log file w/full results from RT suite run (if applicable).
+   - [ ] Commit 'test_changes.list' (unless file is empty). 
```

Comment:
@gspetro-NOAA I didn't look carefully at this change to the ``test_changes.list`` info. 

Say the previous PR changed lots of baselines. Now your PR comes along and changes none of those baselines. Your change list will be empty, which is what we want. If you leave the ``test_changes.list`` from the previous PR (which *did* change baselines), someone going back to see the history will be left w/ the idea that your non-baseline changing PR changed those same baselines.

We actually need an empty ``test_changes.list`` on the non-baseline changing PR to indicate that it was in fact non-baseline changing.

Reply:
That's a good point. I'll clarify that in the next PR I make. In the past, people generally just didn't check off this box if they didn't have baseline changes, so then it looked like they hadn't completed all tasks. They may not need to commit a test_changes.list file (e.g., if the previous PR also had no baselines), but the checklist needs to encourage people to take a look and ensure that the file is in proper condition. Best to be explicit, I think. How's this:

- [ ] Verify that 'test_changes.list' indicates which tests, if any, are changed by this PR. If needed, commit the updated 'test_changes.list' file.

Reply:
[ ] Verify that 'test_changes.list' indicates which tests, if any, are changed by this PR. This must be committed, even if it is empty.

Reply:
We should change the code to allow a hash to be added to the top of test_changes.list so no matter what an updated test_changes.list would be able to be committed.

Reply:
Ok, I'll put in Brian's suggested wording then. I agree that adding a hash to the file is likely the best workaround.